### ISSUE TITLE
ClassNotFoundException

### ISSUE THREAD

**Issue by dsdebastiani** [SchibstedSpain/Leku#128]
## Description

Some classes dependencies did't find: `pl.charmas.android.reactivelocation2.ReactiveLocationProvider`.

## Info Required

- Which version of the library do you actually use?
Leku: `3.6.1`
PlayServices: `11.6.0`
- places
- location

- Do you have the localization permission granted?
`yes`

- Are you sending parameters to the activity through the bundle?
```
val intent = LocationPickerActivity.Builder()
                .withLocation(LocationPreference.locationLatitude.toDouble(), LocationPreference.locationLongitude.toDouble())
                .withGeolocApiKey(activity?.getString(R.string.google_geo_api_key))
                .withSearchZone("pt_BR")
                .withSatelliteViewHidden()
                .build(activity)
        activity?.startActivityForResult(intent, Constants.IntentCodes.LOCATION_PICKER)
```

- Android monitor shows any log related to the library when the error is shown?
```
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: br.com.samples.location, PID: 5030
                  java.lang.NoClassDefFoundError: Failed resolution of: Lpl/charmas/android/reactivelocation2/ReactiveLocationProvider;
                      at com.schibstedspain.leku.LocationPickerActivity.setUpMainVariables(LocationPickerActivity.java:166)
                      at com.schibstedspain.leku.LocationPickerActivity.onCreate(LocationPickerActivity.java:140)
                      at android.app.Activity.performCreate(Activity.java:6679)
                      at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
                      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2618)
                      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
                      at android.app.ActivityThread.-wrap12(ActivityThread.java)
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
                      at android.os.Handler.dispatchMessage(Handler.java:102)
                      at android.os.Looper.loop(Looper.java:154)
                      at android.app.ActivityThread.main(ActivityThread.java:6119)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
                   Caused by: java.lang.ClassNotFoundException: Didn't find class "pl.charmas.android.reactivelocation2.ReactiveLocationProvider" on path: DexPathList[[zip file "/data/app/br.com.samples.location-2/base.apk", zip file "/data/app/br.com.samples.location-2/split_lib_dependencies_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_0_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_1_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_2_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_3_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_4_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_5_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_6_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_7_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_8_apk.apk", zip file "/data/app/br.com.samples.location-2/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/br.com.samples.location-2/lib/x86, /data/app/br.com.samples.location-2/base.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_dependencies_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_0_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_1_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_2_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_3_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_4_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_5_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_6_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_7_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_8_apk.apk!/lib/x86, /data/app/br.com.samples.location-2/split_lib_slice_9_apk.apk!/lib/x86, /system/lib, /vendor/lib]]
                      at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
                      at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
                      at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
                      at com.schibstedspain.leku.LocationPickerActivity.setUpMainVariables(LocationPickerActivity.java:166) 
                      at com.schibstedspain.leku.LocationPickerActivity.onCreate(LocationPickerActivity.java:140) 
                      at android.app.Activity.performCreate(Activity.java:6679) 
                      at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118) 
                      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2618) 
                      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726) 
                      at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477) 
                      at android.os.Handler.dispatchMessage(Handler.java:102) 
                      at android.os.Looper.loop(Looper.java:154) 
                      at android.app.ActivityThread.main(ActivityThread.java:6119) 
                      at java.lang.reflect.Method.invoke(Native Method) 
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886) 
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776) 
W/DynamiteModule: Local module descriptor class for com.google.android.gms.googlecertificates not found.
I/DynamiteModule: Considering local module com.google.android.gms.googlecertificates:0 and remote module com.google.android.gms.googlecertificates:0
E/GoogleCertificates: Failed to load com.google.android.gms.googlecertificates
                      pt: No acceptable module found. Local version is 0 and remote version is 0.
                          at com.google.android.gms.dynamite.DynamiteModule.a(:com.google.android.gms.DynamiteModulesB@11743470:11)
                          at hx.a(:com.google.android.gms.DynamiteModulesB@11743470:12)
                          at hx.a(:com.google.android.gms.DynamiteModulesB@11743470:31)
                          at hx.b(:com.google.android.gms.DynamiteModulesB@11743470:30)
                          at ii.a(:com.google.android.gms.DynamiteModulesB@11743470:35)
                          at ii.a(:com.google.android.gms.DynamiteModulesB@11743470:20)
                          at com.google.maps.api.android.lib6.impl.ew.a(:com.google.android.gms.DynamiteModulesB@11743470:141)
                          at com.google.maps.api.android.lib6.impl.fc.run(:com.google.android.gms.DynamiteModulesB@11743470:27)
                          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
                          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
                          at java.lang.Thread.run(Thread.java:761)
```

**Comment by ferranpons** [comment]
@dsdebastiani is it happening on Release or Debug version of your app?
Are you using the new D8 Dexer? 
If you are using the new D8 Dexer, have you tried to disable the D8 dexer? (using `android.enableD8=false` in your `gradle.properties` file)

**Comment by ferranpons** [comment]
@dsdebastiani it's solved in the version 3.6.2 of the library.

If after updating the library it still happen please reopen the issue.

Many thanks and sorry for the inconvenience. 😉

**Comment by bunnysAndroid** [comment]
Im getting same issue in leku:4.0.0
